doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -228,7 +226,7 @@ Simulation in CLI
             ...
         )
 
-- Run |runsimcli_link|_ in CLI and point to the ``server_app`` / ``client_app`` object in the
+- Run :code:`flower-simulation` in CLI and point to the ``server_app`` / ``client_app`` object in the
```

Comment:
Is there a way to easily link to the `flower-simulation` CLI docs? While writing this, I noticed that `flower-simulation` is not listed in the CLI docs: https://flower.ai/docs/framework/ref-api-cli.html

So how about:
1. We add `flower-simulation` to the CLI docs (new PR)
2. We update this PR to link to it